### PR TITLE
Update pension protection guide

### PR DESCRIPTION
**Pull request by alexkouz**
FCA wasn't spelled out.

### CODE REVIEW

File: content/protection.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ If your employer goes out of business you won’t lose your pension pot. This is
 
 ## If your provider can’t pay
 
-You can be compensated for up to 100% of the value of your pot if your pension provider can’t pay you and is authorised by the FCA.
+You can be compensated for up to 100% of the value of your pot if your pension provider can’t pay you and is authorised by the Financial Conduct Authority (FCA).
```
Comment (benlovell) [inline content/protection.md]: @benbarnett Do we use `abbr` for this?

Comment (benbarnett) [inline content/protection.md]: Yep, good spot. I can't remember if there's markdown syntax for this. Will check first thing Monday.

Comment (benbarnett) [inline content/protection.md]: Thinking about this, we don't actually use `abbr` tags elsewhere on the site. Perhaps we can do revisit in a separate PR for all abbreviations. So this is cool as is.
